src/ert/run_models/everest_run_model.py

```diff
@@ -775,11 +781,9 @@ def add(
         )
 
     def get(
-        self, realization_id: int, controls: NDArray[np.float64]
+        self, realization: int, controls: NDArray[np.float64]
```

Comment:
Because there was some confusion regarding this, would perhaps to add a docstring to this to describe what realization means here?

Reply:
^agree

Reply:
Done